index.html

```diff
@@ -444,6 +444,7 @@ <h2 class="title-contact">Contact me</h2>
           <button type="submit" class="button1 form-button">
             Get in Touch
           </button>
+          <small id="small1">Error message</small>
```

Comment:
- [ ] I think it would be better to put the error message in `Red` so that user will be well alerted about your input validation